cl/alerts/templates/alert_email_es.txt

```diff
@@ -27,7 +27,7 @@ Disable this Alert (one click): https://www.courtlistener.com{% url "disable_ale
 ************************
 This alert brought to you by the 501(c)(3) non-profit Free Law Project
  - Blog: https://free.law
- - Twitter: http://twitter.com/freelawproject
+ - Twitter: https://twitter.com/freelawproject
```

Comment:
are you ready to switch to x.com

Reply:
As you wish. I just pushed changes to remove Twitter from most places, and add BlueSky in a few. We need new icons, but that feels out of scope.